Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM registry.access.redhat.com/ubi8/ubi
 
 ARG conftest_version=0.33.2
 ARG BATS_VERSION=1.6.0
+ARG cyclonedx_version=0.24.0
```

Comment:
Actually I wanted to get the newest tag from https://github.com/CycloneDX/cyclonedx-cli/tags and set it to `cyclonedx_version`  so we don't need to update the version but have not found a quick way to do it.

Reply:
I think that we should be fine using a fixed version for now, and we can consider making periodic updates to our tooling versions in the future.
About the current version - would it be better to use the newest [v0.24.2](https://github.com/CycloneDX/cyclonedx-cli/releases/tag/v0.24.2) version here?

Reply:
Ok, thanks!